src/db.rs

```diff
@@ -444,6 +447,46 @@ pub async fn get_active_app_infos_since(
         .map(|app| app.into_iter().map(|a| a.into()).collect())
 }
 
+pub async fn get_inactive_app_infos_since(
+    c: &mut SqliteConnection,
+    serial: u32,
+) -> sqlx::Result<Vec<AppInfo>> {
+    sqlx::query_as::<_, DBAppInfo>("SELECT * FROM app_infos WHERE active = false AND serial > ?")
+        .bind(serial)
+        .fetch_all(c)
+        .await
+        .map(|app| app.into_iter().map(|a| a.into()).collect())
+}
+
+pub async fn app_info_exists(c: &mut SqliteConnection, app_id: &str) -> sqlx::Result<bool> {
```

Comment:
```suggestion
pub async fn app_exists(c: &mut SqliteConnection, app_id: &str) -> sqlx::Result<bool> {
```

Reply:
This is supposed to be app_info actually.